internal/cli/opsmanager/agents/upgrade.go

```diff
@@ -29,6 +29,8 @@ type UpgradeOpts struct {
 	store store.AgentUpgrader
 }
 
+var upgradeTemplate = "Updating agent versions to the latest available."
```

Comment:
[nit] missing new line
```suggestion
var upgradeTemplate = "Updating agent versions to the latest available.\n"
```